### PR TITLE
Bump Network and Netty dependencies

### PR DESCRIPTION
**Pull request by Camotoy**

### CODE REVIEW

**Review by rtm516 — APPROVED** [review]
Works with no problem on Windows 10 1909

**Comment by MacacoLew** [comment]
Merged (?

**Comment by Camotoy** [comment]
Not yet. Someone reported connections being worse with this PR.

File: connector/pom.xml [hunk 1]
```diff
@@ -10,6 +10,10 @@
     </parent>
     <artifactId>connector</artifactId>
 
+    <properties>
+        <netty.version>4.1.56.Final</netty.version>
```
Comment (Tim203) [inline connector/pom.xml:14]: This version is outdated now. The newest version is 4.1.58.Final

Comment (Camotoy) [reply connector/pom.xml:14]: Thanks for the heads-up!

Comment (toinouH) [reply connector/pom.xml:14]: https://github.com/netty/netty/releases/tag/netty-4.1.59.Final
This is now 4.0.59.Final :)

**Review by lukeeey — APPROVED** [review]

**Pull request merged**